packages/plugin-dom-layout/src/ActionableGroupDomObjectRenderer.ts

```diff
@@ -36,4 +53,27 @@ export class ActionableGroupDomObjectRenderer extends NodeRenderer<DomObject> {
         };
         return objectGroup;
     }
+    /**
+     * Update option rendering after the command if the value of visible
+     * changed.
+     *
+     * @param actionable
+     * @param element
+     */
+    protected _updateActionables(): void {
+        for (const [actionable, element] of this.actionableNodes) {
```

Comment:
I don't understand. Why do we have two "_updateActionables" functions?

Reply:
We have three :sunglasses:
One for Actionable, one for ActionableGroup, and one for ActionableGroupSelectItem.

Reply:
But... only the one of Actionable shoulb be called `_updateActionables`, right ? Shouldn't the others be `_updateActionableGroupItems` and `_updateActionableGroupeSelectItems` ?